src/Microsoft.AspNetCore.Razor.Language/Intermediate/CSharpExpressionAttributeValueIRNode.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Razor.Language.Intermediate
+{
+    public sealed class CSharpExpressionAttributeValueIRNode : RazorIRNode
+    {
+        public override ItemCollection Annotations => ReadOnlyItemCollection.Empty;
```

Comment:
Should be lazy about annotations. The purpose of these is to enable others to add metadata to our IR nodes.

```C#
private ItemCollection _annotations;

public override ItemCollection Annotations
{
    get
    {
        if (_annotations == null)
        {
            _annotations = new DefaultItemCollection();
        }

        return _annotations;
    }
}
```